lonestar/analytics/distributed/README.md

```diff
@@ -114,26 +119,118 @@ Heterogeneous apps have additional command line parameters:
 
 Specifies the total number of PHYSICAL machines on the system. For example,
 you could have 2 machines with 8 GPUs each for a total of 16 processes,
-but you still would only have 2 machines. Therefore, you would use 
+but you still would only have 2 machines. Therefore, you would use
 `-num_nodes=2`. Note that there **must** be one process per GPU in use.
 
 `-pset=<string>`
 
-Specifies the architecture to run on on a single machine using "c" (CPUs) and 
-"g" (GPUs). For example, if I have 2 machines with 8 GPUs each, 
-but I want to run with 3 GPUs on each machine, I would use `-pset="ggg"`. 
-Therefore, combined with `-num_nodes=2`, I would have a total of 6 units of 
+Specifies the architecture to run on on a single machine using "c" (CPUs) and
+"g" (GPUs). For example, if I have 2 machines with 8 GPUs each,
+but I want to run with 3 GPUs on each machine, I would use `-pset="ggg"`.
+Therefore, combined with `-num_nodes=2`, I would have a total of 6 units of
 execution: 3 GPUs on 2 machines for a total of 6. This creates a total of
 6 processes across the 2 machines (1 for each GPU).
 
-Also, it suffices to use only one "c" in pset to run on CPUs on your machines: 
-you can specify the amount of CPUs to use using the aforementioned thread 
-option `-t`.
+Also, it suffices to use only one "c" in pset to run on CPUs on your machines:
+you can specify the amount of cores/hyperthreads to use using the
+aforementioned thread option `-t`.
+
+Examples for Running Provided Apps
+================================================================================
+
+To run 3 processes all on a single machine, use the following:
+`mpirun -n=3 ./<app> <input-graph> -t=<num-threads-per-process>`
+
+To run on 3 CPUs on h1, h2, and h3, use the following:
+`mpirun -n=3 -hosts=h1,h2,h3 ./<app> <input-graph> -t=<num-threads>`
```

Comment:
I have not used the "hosts" option, so I don't know if it is correct.

Reply:
I used to use it a long time ago on our servers to debug things; it should work.